File: go/iq/fb-thrift_iq~get~linked_accounts.go
// BuildBizLinkingGetLinkedAccounts builds <iq type="get" xmlns="fb:thrift_iq">. Replies with one of: Success, Forbidden, Error.
func BuildBizLinkingGetLinkedAccounts(smaxId int, from types.JID) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"xmlns": "fb:thrift_iq", "smax_id": smaxId, "from": from, "to": "s.whatsapp.net", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "linked_accounts", Attrs: waBinary.Attrs{}},
		},
	}
}

File: rust/iq/fb-thrift_iq~get~linked_accounts.rs
/// <iq type="get" xmlns="fb:thrift_iq">. Replies with one of: Success, Forbidden, Error.
pub fn build_biz_linking_get_linked_accounts(smaxId: i64, from: Jid) -> Node {
    NodeBuilder::new("iq")
        .attr("xmlns", "fb:thrift_iq")
        .attr("smax_id", smaxId)
        .attr("from", from)
        .attr("to", "s.whatsapp.net")
        .attr("type", "get")
        .child(NodeBuilder::new("linked_accounts").build())
        .build()
}